src/Organisms/EnhancedList/EnhancedList.jsx

```diff
@@ -66,47 +71,40 @@ const EnhancedList = ({
         <Pagination/>
         {onSearch != null &&
           <SearchInput
+            size="small"
             value={searchTerm}
             onChange={event => setSearchTerm(event.target.value)}
             endAdornment={
               <InputAdornment>
-                <IconButton data-testid="search-button" onClick={() => onSearch(searchTerm)}>
-                  <Search size={18}/>
-                </IconButton>
+                <Icon Component={Search} data-testid="search-button" onClick={() => onSearch(searchTerm)} />

```

Comment:
Do we really need those testids ?
Icons should have an alt text and then you can use getByAltText in your tests.

Reply:
Our icons are not images, so they can't have `alt` attributes (they usually are `button` or `svg`).

Reply:
Damn, to bad